src/behaviors/days-until/days-until.js

```diff
@@ -101,21 +101,22 @@ class DaysUntil extends Behavior {
       }
 
       topic = topic.replace(/\d+ days? until BronyCon!/gi, message);
+      console.log(topic);
```

Comment:
Please go back to `bot.log` for these. There is a long term desire to want to explicitly log out to a file via the bot that I would like to implement and keeping the logging to the bot would help with that.

Reply:
We weren't sure it was functioning in those cases; in fact, those were never called anyway. It seems like `Promise.catch` is more reliable in these cases, and was used for quite a bit of the debugging around there. (Turns out you can't set the topic of a channel you're not in--but you can certainly post there! :shrug:)